packages/hardhat-waffle/src/waffle-chai.ts

```diff
@@ -64,17 +64,19 @@ export function waffleChai(chai: Chai.ChaiStatic, utils: Chai.ChaiUtils) {
 }
 
 function supportCalledOnContract(Assertion: Chai.AssertionStatic) {
+  // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I'm not sure about this. Shouldn't `chai` be a peer dependency of `hardhat-waffle`?

Reply:
I think this is meant to import the version installed by `ethereum-waffle`